pandunia-loge.csv

```diff
@@ -3287,12 +3296,15 @@ tanur|tanur|||fas:heb:tanur, ara:(tannūr), eng:tandoor, athanor, tur:tandır, h
 Tanzania|Tanzania|desh|TZ||Tanzania||Tanzania|||||||||||||Tanzanio|Tansania|Tanzania
 tapa|tapa||||plug (stop with a plug)||taponar||||||||||||||korkittaa (tukkia)|zatkać, ztykać
 tape|tape|||spa:tur:tapa, por:tampa, ell:τάπα (tapa), bul:тапа (tapa), eng:tap|stopper (cap, peg, tap)||tapón (tapa)||||||||||||||tulppa (tappi, korkki)|korek, zatyczka, szpunt
-tarif|tarif|||eng:tariff, spa:por:tarifa, deu:fra:tur:tarif, ara:(taʿrīf), rus:тариф (tarif), hin:टैरिफ़ (ṭairif)|tarif (tax, duty)||impuesto (tributo, tarifa)||||||||||||||taksa (tariffi)|opłata, taryfa
-tarja|tarja||||translate (interpret)||traducir (interpretar)|||||||||||||traduki|tulkata (kääntää)|przetłumaczyć, tłumaczyć, interpretować
-tarje|tarje|||ara:(tarjama), hin:तर्जुमा (tarjuma), urd:(tarjuma), ben:তর্জমা (tôrjôma), may:terjemah, tur:tercüme|translation (interpretation)|traduction|tradicción (interpretación)||перевод||翻译|||||||||traduko|käännös (tulkinta)|tłumaczenie, interpretacja
-tarjer|tarjer||||interpreter (translator)||traductor|||||||||||||tradukisto|tulkki (kääntäjä)|tłumacz
-tas|tas|||rus:таз, ara:طشت‎ (ṭašt)|dish||plato||блюдо (таз)|||||||||||plado|astia (kulho)|danie
-tate|tate|||hin:तट (taṭ), ben:তট (taṭ)|shore (bank, edge)|bord (rive)|orilla (ribera)|margem|||岸|岸||||||||||wybrzeże (brzeg, krawędź)
+tarif:|tarif|||eng:tariff, spa:por:tarifa, deu:fra:tur:tarif, ara:(taʿrīf), rus:тариф (tarif), hin:टैरिफ़ (ṭairif)|tarif (tax, duty)||impuesto (tributo, tarifa)||||||||||||||taksa (tariffi)|opłata, taryfa
+tarj:a|tarja||||translate (interpret)||traducir (interpretar)|||||||||||||traduki|tulkata (kääntää)|przetłumaczyć, tłumaczyć, interpretować
+tarj:e|tarje|||ara:(tarjama), hin:तर्जुमा (tarjuma), urd:(tarjuma), ben:তর্জমা (tôrjôma), may:terjemah, tur:tercüme|translation (interpretation)|traduction|tradicción (interpretación)||перевод||翻译|||||||||traduko|käännös (tulkinta)|tłumaczenie, interpretacja
+tarj.er:|tarjer||||interpreter (translator)||traductor|||||||||||||tradukisto|tulkki (kääntäjä)|tłumacz
+tartib:a|tartiba||||order (arrange)||ordenar (arreglar)||||整理|揃える|||व्यवस्थित करना||||||järjestää (järjestellä)|porządkować, organizować
+tartib:e|tartibe|||ara:تَرْتِيب‎ (tartīb), hin:तरतीब (taratība), swa:taratibu, may:tatatertib, tur:tertip|order (arrangement, sequence)|ordre|orden|ordem|установка||常规 (次序)|順|||||tatatertib|taratibu|||järjestys|porządek
```

Comment:
Pandunia aims to borrow word roots and the triliteral Arabic root is r-t-b. ta- in "tartib" is a prefix. Sometimes a derived word has spread more wide than a simpler form. That is true in the case of "tartib" (order, arrangement) and "tasvir" (image, picture), for example. However the root is preferred and it can be easily back-derived by those whose language has borrowed extensively from Arabic. For example, Turkish has borrowed "rütbe" (rank) and "tertip" (order) from this Arabic root.

I propose this etymology:
ara: رَتَّبَ (rattaba), swa:ratibu, hau:rattaba, fas: ترتيب (tartīb), hin:तरतीब (taratība), may:tertib, tur:tertip.